src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx

```diff
@@ -68,35 +71,47 @@ export const ActionSelectionForm = ({
           <div className="col-12 col-lg">
             <AssetInformationPanel tokenId={tokenId} tokenRegistryAddress={tokenRegistryAddress} />
           </div>
-          <div className="col-12 col-lg">
-            <EditableAssetTitle role="Beneficiary" value={beneficiary} isEditable={false} />
-          </div>
-          <div className="col-12 col-lg">
-            <EditableAssetTitle role="Holder" value={holder} isEditable={false} />
-          </div>
+          {isEBLSurrendered ? (
+            <div className="col-12 col-lg-auto align-self-end">
+              <div className="py-3">
+                <TagBorderedSurrendered>Surrendered</TagBorderedSurrendered>
```

Comment:
```suggestion
                <TagRedBordered>Surrendered</TagRedBordered>
```
Since the "Surrendered" word is passed in, then the component itself is not specific to surrendered, will be better to give it a more generic naming